test/Help.Tests.ps1

```diff
@@ -0,0 +1,188 @@
+<#	
+	.NOTES
+		===========================================================================
+		Created with: 	SAPIEN Technologies, Inc., PowerShell Studio 2016 v5.2.119
+		Created on:   	4/12/2016 1:11 PM
+		Created by:   	June Blender
+		Organization: 	SAPIEN Technologies, Inc
+		Filename:		*.Help.Tests.ps1
+		===========================================================================
+	.DESCRIPTION
+	To test help for the commands in a module, place this file in the module folder.
+	To test any module from any path, use https://github.com/juneb/PesterTDD/Module.Help.Tests.ps1
+#>
```

Comment:
Yep, Plaster passes this test set, as all of the public functions have the main help set :) The original tests failed on -WhatIf and -Confirm not having help, but since those are essentially standard switches, I excluded them, as I'm not sure how much help documenting those switches would be.